pygeoapi/process/manager/postgresdb_.py

```diff
@@ -0,0 +1,261 @@
+# =================================================================
+#
+# Authors: Francesco Martinelli <francesco.martinelli@ingv.it>
+#
+# Copyright (c) 2024 Francesco Martinelli
+#
+# Permission is hereby granted, free of charge, to any person
+# obtaining a copy of this software and associated documentation
+# files (the "Software"), to deal in the Software without
+# restriction, including without limitation the rights to use,
+# copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following
+# conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+# OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+# HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+# WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# =================================================================
+
+# Requires postgresql database structure.
+# Import dump:
+# psql -U postgres -h 127.0.0.1 -p 5432 postgres <
+#   tests/data/postgres_manager_full_structure.backup.sql
+
+import json
```

Comment:
Imports should be in the order of standard library, third party packages, and local imports, alphabetically, grouped by spaces.  So:


```python
import json
import logging
from pathlib import Path
from typing import Any, Tuple

import psycopg2
import psycopg2.extras

from pygeoapi.process.manager.base import BaseManager
from pygeoapi.process.base import (
    JobNotFoundError,
    JobResultNotFoundError,
    ProcessorGenericError
)
from pygeoapi.util import JobStatus
```